Source/SwiftLintFramework/Extensions/Array+SwiftLint.swift

```diff
@@ -24,3 +24,27 @@ extension Array where Element: NSTextCheckingResult {
         return map { $0.range }
     }
 }
+
+extension Array where Element: Equatable {
+    var unique: [Element] {
+        var uniqueValues: [Element] = []
+        forEach { item in
+            if !uniqueValues.contains(item) {
+                uniqueValues += [item]
+            }
+        }
+        return uniqueValues
+    }
+}
+
+public extension SequenceType {
```

Comment:
Right, I copy-pasted it from my other project, doesn't make any sense for this to be public.